docs/use-cases/deploy-ml.md

```diff
@@ -3,6 +3,63 @@ title: "Train and deploy image classification models"
 linkTitle: "Train and deploy classification models"
 weight: 50
 type: "docs"
-layout: "empty"
-canonical: "ml/"
+tags: ["data management", "data", "services"]
+no_list: true
+description: "Use Viam's machine learning capabilities to train image classification models and deploy these models to your machines."
+image: "/ml/training.png"
+imageAlt: "Machine Learning"
+images: ["/ml/training.png"]
 ---
+
+Deploy an image classification model onto your machine with Viam's machine learning (ML) service.
+Do this all on one platform with Viam's data management service: collect data, create a dataset and label it, train or upload an ML model, and deploy the model.
+Then, test if your model works for classifying objects in a camera stream or existing images with the `mlmodel` classification model of vision service.
+
+<table>
+  <tr>
+    <th>{{<imgproc src="/ml/collect.svg" class="fill alignright" style="max-width: 300px" declaredimensions=true alt="Collect data">}}
+      <b>1. Collect</b>
+      <p>Start by collecting data from your cameras, sensors, or any other source on your machine with the <a href="/data/">data management service</a>. You can <a href="/data/view/">view the data</a> on the <b>Data tab</b>.</p>
```

Comment:
So this is supposed to be specifically for image classification. I don't think sensors or other sources are needed
```suggestion
      <p>Start by collecting data from your cameras with the <a href="/data/">data management service</a>. You can <a href="/data/view/">view the data</a> on the <b>Data tab</b>.</p>
```

Reply:
Makes sense, thank you!